package.json

```diff
@@ -36,20 +36,20 @@
     "node": ">= 8.0.0"
   },
   "dependencies": {
-    "debug": "4.1.1",
-    "glob": "7.1.6",
-    "lodash": "4.17.20",
+    "debug": "4.3.1",
+    "glob": "7.1.7",
+    "lodash": "4.17.21",
```

Comment:
Given the frequency of vulnerabilities in lodash, how would you feel about leaving this dependency floating? That would allow clients of your library to fix it in their repos (via `npm audit fix` or so) while you work on spinning a new release.

```suggestion
    "lodash": "^4.17.21",
```